### ISSUE TITLE
waiting for 1.21

### ISSUE THREAD

**Issue by SnowIves** [NimajnebEC/auto-elytra#18]
Really need the 1.21 update for this mod. simply change the version number （1.20.X to 1.21.X）does not work.

**Comment by Ikiroux** [comment]
I try to change the mod version to 1.20.x to 1.21.x, cloth config version and mod menu version, this does'nt work.
I hope that the mod will be ported to 1.21.x, it's objectively the most useful mod that I have ever used.

**Comment by brooke-ec** [comment]
Yeah, the addition of structured components in 1.20.5 requires a non-insignificant rewrite of parts of this mod. Nevertheless I will hopefully have a 1.21 version done soon!

**Comment by Ikiroux** [comment]
.

**Comment by SnowIves** [comment]
> Yeah, the addition of structured components in 1.20.5 requires a non-insignificant rewrite of parts of this mod. Nevertheless I will hopefully have a 1.21 version done soon!

But you haven't finished it until today.

**Comment by Ikiroux** [comment]
I found another mod that does the exact same thing. JJElytraSwap if you want.

**Comment by brooke-ec** [comment]
Yeah, definitely use [JJElytraSwap](https://modrinth.com/mod/jjelytraswap) as it looks very good!

If you need a Forge/NeoForge version I will still endeavour to update AutoElytra but I'm sorry, I can't say when this will be completed

**Comment by BlockMiner2k** [comment]
Theres also this one https://modrinth.com/mod/auto_elytra
